src/algebra/symmetrized.lean

```diff
@@ -0,0 +1,204 @@
+/-
+Copyright (c) 2021 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import algebra.module.basic
+import tactic.abel
+
+/-!
+# Symmetrized algebra
+
+A commutative multiplication on a real or complex space can be constructed from any multiplication
+by "symmetrization" i.e
+$$
+a \circ b = \frac{1}{2}(ab + ba)
+$$
+
+## Implementation notes
+
+The approach taken here is inspired by algebra.opposites. We use Oxford Spellings
+(IETF en-GB-oxendict).
```

Comment:
I don't think we need to mention this, since none of the lemma names use either the `z` or the `s`.